src/styles/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: flex;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-family: "Roboto", sans-serif;
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/b1acffc7-3a87-4bf3-84bd-cdd12d5420d0)
You can also set the font for the whole page and inherit where needed